src/Grok.Net.Tests/Patterns/grok-custom-patterns

```diff
@@ -1 +1,3 @@
-﻿ZIPCODE [1-9]{1}[0-9]{2}\s{0,1}[0-9]{3}
\ No newline at end of file
+﻿ZIPCODE [1-9]{1}[0-9]{2}\s{0,1}[0-9]{3}
+WRONGPATTERN1 \\\\\
+WRONGPATTERN2 \\\\\
```

Comment:
Does it make sense to add two the same cases? Why not
```
WRONGPATTERN1 \\\\\
WRONGPATTERN2
WRON . //
\\\
```
or something like that

Reply:
Yes but then wouldn't it throw a `FormatException` for each test? due to these lines:
```cs
            string[] strArray = line.Split(new[] { ' ' }, 2);
            if (strArray.Length != 2)
            {
                throw new FormatException("Custom pattern was not in a correct form");
            }
```

Reply:
Yeah. Perhaps we should add another file with the wrong patterns but then we need a feature to specify what file should be loaded.
Good point for improvement, to let client add a file from any directory instead of hardcoded `Patterns`